app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -279,9 +254,13 @@ fun SpotCard(
               .height(120.dp)
               .shadow(8.dp, shape = MaterialTheme.shapes.medium)
               .padding(4.dp)
-              .clickable(onClick = { navigationActions.navigateTo(Screen.CARD) })
+              .clickable(
+                  onClick = {
```

Comment:
I would think that it might be preferable to first select the parking before navigating to Card screen since it might be asynchronous? It might lead to a situation where the card screen loaded before we selected the parking?

Reply:
Good point! Will do